pkg/converters/ingress/annotations/backend_test.go

```diff
@@ -1209,6 +1209,7 @@ const (
 )
 
 var corsDefaultOrigin = []string{"*"}
+var corsDefaultOriginRegex = []string{}
```

Comment:
Now the default value is nil instead of an empty slice, you can remove this declaration and all the items where it's used.

Reply:
ok, removed